command/server/listener.go

```diff
@@ -97,6 +97,15 @@ func listenerWrapTLS(
 		}
 		tlsConf.PreferServerCipherSuites = preferServer
 	}
+	if v, ok := config["tls_require_valid_client_cert"]; ok {
```

Comment:
Ah, I see. Since we are picking the option out of the lot, I guess its okay to diverge from the 1:1 naming convention between tlsConf and listener configs. I agree with making it a bool; makes it a lot easier for users to just enable as opposed to looking up for valid values. `require_and_verify_client_cert` seems perfect; atleast we are making it consistent with the our choice of option :-)